webrtc-stats.html

```diff
@@ -2989,6 +2989,19 @@ <h2>
                   addresses, and fully qualified domain names (FQDNs). See
                   [[!RFC5245]] section 15.1 for details.
                 </p>
+                <p>
+                  A user agent should make sure to limit exposure of remote candidate addresses to
+                  the ones already known to the web application, in particular peer reflexive remote
```

Comment:
The test actually makes it clear that it's not "the ones already known to the web application", it's "the ones already known to be in use on this peerconnection". In the test, the web application knows all the addresses (from the "other" PC), but it doesn't "know" that one PC's local address is in use by the other PC as a remote address.

Suggested rephrase:

"An user agent should make sure that only remote candidate addresses that the web application has configured on the peerconnection are exposed; in particular peer reflexive remote canididate addresses that don't match a candidate set by addIceCandidate."

Reply:
I initially thought that keeping some flexibility in the way implementations would handle that could be good.
The added test is somehow a degenerated case since the web page already learned the IP address.

I agree though that such smarter implementation would be more complex, will not provide a lot of benefit.
It would also make the test not fully authoritative. Let's mandate it then.